packages/pwa-kit-react-sdk/src/ssr/server/express.js

```diff
@@ -212,6 +212,13 @@ export const createApp = (options) => {
 
     // Attach built in routes and middleware
 
+    // Attach the application configuration to the request object.
+    app.use((req, _, next) => {
+        req.config = options.mobify
+
+        next()
+    })
```

Comment:
here we are attaching configs at the request level, by using a middleware. I'm wondering could we do it at the express app level like:

```suggestion
    app.config = options.mobify
```

Reply:
We should only load the config once, at server boot up time.

Reply:
Totally agree. I'll make this change